check_diff/src/main.rs

```diff
@@ -15,11 +20,36 @@ struct CliInputs {
     rustfmt_config: Option<Vec<String>>,
 }
 
-fn main() {
-    let args = CliInputs::parse();
+/// Clone a git repository and cd into it.
+///
+/// Parameters:
+/// url: git clone url
+/// dest: directory where the repo should be cloned
+fn clone_git_repo(url: &str, dest: &str) {
+    env::set_var("GIT_TERMINAL_PROMPT", "0");
```

Comment:
[Just a heads up `env::set_var` is going to be made unstable in rust 2024](https://github.com/rust-lang/rust/pull/124636). It's probably better to set the `GIT_TERMINAL_PROMT` in this case using the [`Command::env`](https://doc.rust-lang.org/std/process/struct.Command.html#method.env) builder method.